gr-digital/lib/ofdm_cyclic_prefixer_impl.cc

```diff
@@ -31,135 +31,147 @@ namespace gr {
   namespace digital {
 
     ofdm_cyclic_prefixer::sptr
-    ofdm_cyclic_prefixer::make(size_t input_size, size_t output_size, int rolloff_len, const std::string &len_tag_key)
+    ofdm_cyclic_prefixer::make(int fft_len, std::vector<int> cp_lengths, int rolloff_len, const std::string &len_tag_key)
```

Comment:
Your overloaded make() call could simply create a vector of length 1 and then we don't need to overload the ctors, too.